src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js

```diff
@@ -178,7 +189,7 @@ const AddProposalPanel = React.memo(({ setProposalData }) => {
     }
 
     if (link && !forumRegex.test(link)) {
-      errors.push('Forum post link not provided ')
+      errors.push('Forum post link not provided or not valid')
```

Comment:
Make sense that addition?

Reply:
We should stop using the term "Forum post" and start using "Proposal information". 

This should then read "Proposal information link not provided".

Reply:
It mislead the user, Say its not provided while the text input have alot text, have sure @willjgriff ?

Reply:
Fixed